### ISSUE TITLE
Enhancement of the War day progress

### ISSUE THREAD

**Issue by Eagle334** [AaronTraas/Clash-Royale-Clan-Tools#12]
During the War day, retrieve via the API and display information on the clan's country , its level...
And a must have would be to have the history of the evolution of the positions, like this one :
https://www.racefans.net/wp-content/uploads/2008/10/2008chinesegplapchart.gif
And a quick win : display the winrate of each team.

**Comment by AaronTraas** [comment]
Not all of this is possible.

Displaying opponent clan countries/levels is doable, but for countries I'd need open-source or compatibly licenced flag images. If you can wrangle them for me, that would be awesome.

For the other stuff, that's not necessarily available. Here's a sample what I can get from the opposing clans:

        {
            "tag": "#JY8YVV",
            "name": "Agrassar",
            "badgeId": 16000010,
            "clanScore": 1749,
            "participants": 19,
            "battlesPlayed": 18,
            "wins": 7,
            "crowns": 15,
            "battlesRemaining": 1
        }

I can then query the API for more data about each clan:

{
    "tag": "#JY8YVV",
    "name": "Agrassar",
    "type": "open",
    "description": "Rules, stats, discord link, and info at https://agrassar.com",
    "badgeId": 16000010,
    "clanScore": 38697,
    "clanWarTrophies": 1749,
    "location": {
        "id": 57000006,
        "name": "International",
        "isCountry": false
    },
    "requiredTrophies": 3000,
    "donationsPerWeek": 6516,
    "clanChestStatus": "inactive",
    "clanChestPoints": 1471,
    "clanChestLevel": 9,
    "clanChestMaxLevel": 10,
    "members": 43,
    "memberList": []
}

No way I can derive total win rate or relative positions from that.

Also, in case it isn't obvious, I'm not gathering perpetual history on every clan. There's no sane way to do so. I'd also hit API rate limits at some point, I'm sure.

**Comment by Eagle334** [comment]
Regarding free to use flag : http://www.famfamfam.com/lab/icons/flags/

When i talk about '_the history of the evolution of the positions_', I speak on the second day of the wars.
I have a cron that requests the API every 15 minutes. My proposal is to store the values (or already one per hour to have max 24 values) to see the progress of the teams over the day and not just display the current status. 
Like this one :
https://www.racefans.net/wp-content/uploads/2008/10/2008chinesegplapchart.gif

And finally about '_And a quick win : display the winrate of each team._', it is the same, i speak about the war day. Need to have the number of fights done, and the number of fights won.
At least it's possible with our team information. Not sure it's possible to have the nb of fights done for other teams.
Basically I observe that we have to have at least 50% to be 3rd or better.
